ui.frontend/src/main/webpack/site/js/components/searchForm.ts

```diff
@@ -0,0 +1,137 @@
+import type { SearchCallbacks } from '../types/callbacks'
+import buildLoadMoreButton from './loadMoreButton'
+import buildSearchResult from './searchResults'
+import buildSearchTab, {
+  removeSearchResults,
+  removeSearchTabs,
+  removeSelectedTabFromSearchContainer,
+  Tab,
+} from './searchTabs'
+
+type SearchFormSubmitEventOption = SearchCallbacks
+
+const buildSearchForm = (): HTMLFormElement => {
+  const searchForm = document.createElement('form')
+  searchForm.classList.add('saas-container_form')
+
+  return searchForm
+}
+
+export const triggerSearch = async (
+  searchForm: HTMLFormElement,
+  searchInputElement: HTMLInputElement,
+  searchUrl: string | undefined,
+  searchTabs: string[],
```

Comment:
This property has been changed to a array of objects
sample search config json:
`{
  "title" : "Search Cmp",
  "searchFieldPlaceholderText" : "Type search term here...",
  "searchButtonText" : "Search",
  "loadMoreButtonText" : "Load more",
  "searchTabs" : [ {
    "title" : "Search tab",
    "url" : "/content/saas-aem-module/us/en/_jcr_content/root/container/container/search/search-tabs/searchtab.model.json?q=pg",
    ":type" : "saas-aem-module/components/searchtab"
  }, {
    "title" : "ZIP Only",
    "url" : "/content/saas-aem-module/us/en/_jcr_content/root/container/container/search/search-tabs/searchtab_765476375.model.json?q=pg",
    ":type" : "saas-aem-module/components/searchtab"
  } ],
  "autosuggestUrl" : "/content/saas-aem-module/us/en/_jcr_content/root/container/container/search.autocomplete.json",
  "autocompleteTriggerThreshold" : 3,
  ":type" : "saas-aem-module/components/search",
  ":items" : { },
  ":itemsOrder" : [ ]
}`